appveyor.yml

```diff
@@ -60,18 +60,18 @@ for:
       only:
         - next
     after_test:
-      - iscc /q "Installer Build Script.iss" 
-      - move Installers\Rubberduck.Setup.exe Installers\Rubberduck.Setup.%APPVEYOR_BUILD_VERSION%-pre.exe
+      - iscc /O "Rubberduck.Deployment\InnoSetup\Installer Build Script.iss" "/DConfig=Release"
```

Comment:
Can we make the `Release` here configurable? We use the same keyword in the line 30 so it would be good to make it stay in lockstep to avoid problems should we need to build non-release builds?

Reply:
@bclothier IMO it's best to only package installers for a release build, and leave debug config for local/manual builds. @rubberduck-vba/dev thoughts?

Reply:
Agreed. My goal was more to eliminate hard-coded parameters as much as I can from this new build process which is quite.... rube-goldsberg-esque.... 😄

Reply:
if we really wanted, we might be able to access the build matrix and pass what we passed there to the installer. Not that a debug installer makes terribly much sense...

Reply:
What did you mean by build matrix @Vogel612 ? We are using AppVeyor to confirm build in development that generates an installer with Github Tag ready for deploy.

Reply:
Just to clarify -- I wasn't suggesting that we be able to build non-release builds with AppVeyor; only that we avoid hard-coding the word `Release` in more than one places in the script, to make it more maintainable (?). 

@PeterMTaylor  in local builds, we normally use `Debug` configuration but in this script, it should build using `Release` configuration. In original version it was mentioned only once but now I have to pass it to the compiler so it can find the files it needs (because it needs to look in `bin\Release`, rather than `bin\Debug` as is the case locally). My changes above simply hard-code the word `Release` which is now a potential trap if we in future need to support different release builds.

Reply:
hum... I'd  say YAGNI applies here. I highly doubt we'll need different installers and even if we do, the installer will be created off the release build...